src/app/components/Button.js

```diff
@@ -37,5 +32,14 @@ export const Button = (
     btnDomObj.appendChild(spanElem);
   }
 
+  btnDomObj.setCorrectAnswer = () => {
```

Comment:
Ja dalej uważam, ze to nie powinno być odpowiedzialnością Buttona (zasada Single Responsibility i Open Closed, ogolnie jak sie zmieni cos w quizie to zmieniamy Buttona, który mógłby działać bez quizu - co im przeczy), no ale jest lepiej niż bylo :)